doc/api/console.md

```diff
@@ -240,7 +240,7 @@ undefined
 
 ### console.debug(data[, ...args])
 <!-- YAML
-added: v8.0.0
+added: v9.2.x
```

Comment:
Can you replace `v9.2.x` with `REPLACEME`. The release process will replace it with the correct version.

Reply:
Thanks! Didn't knew that. Done ;)

Reply:
Wouldn't it be more accurate to use `added: v8.0.0` (since it is defined in v8.0.0 and above) and add another "Changes" row for the change made in https://github.com/nodejs/node/pull/17033? I don't think it makes much of a difference, just wanted to bring it up.